sdk/src/main/java/io/radar/sdk/RadarTrackingOptions.kt

```diff
@@ -277,7 +277,13 @@ data class RadarTrackingOptions(
         /**
          * Determines the id of the notification. Optional, defaults to `20160525`.
```

Comment:
What's the significance of this default?

Reply:
it's Radar's birthday.